### DISCUSSION TITLE
Gestion de la complexité

### DISCUSSION THREAD

**Discussion by hrenaud** [cnumr/best-practices-wordpress#7] [General] [answered]
Nous avons échangé sur la discussion concernant les métadatas que l'information `complexity_implementation` venant des **115 bonnes bonnes pratiques** était difficile à mettre en œuvres sur ce référentiel, car nous avons décidé d'avoir plusieurs publics, ayant des connaissances techniques différentes.

Le personna `Code(use·ur) → Développeu·se·r` pourra modifier le code et le déployer en FTP, alors qu'un utilisateur `No-code → Madame et Monsieur tout le monde` ne sera pas le faire sans passer par un plugin.

Je propose donc d'ouvrir une discussion pour voir si on fait une nouvelle façon d'afficher cette information du type comme ci-dessous, hors des metadatas ou dedans.

|Personna|complexity_implementation|
|---|---|
|`Code(use·ur) → Développeu·se·r`|`Facile 🐣`|
|`Low-code → Freelance et développeur Front-End d'agences`|`Standard 😁`|
|`Designeu(se·r)`|`Difficile 😅`|
|`No-code → Madame et Monsieur tout le monde`|`Difficile 😅`|

**Comment by dNicolle** [comment, marked as answer]
[Suivi de cette discussion dans celle sur la mise à jour du template](https://github.com/cnumr/fiches-wordpress-greenit/discussions/39)
